interfaces/Portalicious/src/app/pages/project-registration-activity-log/components/activity-log-expanded-row/activity-log-expanded-row.component.ts

```diff
@@ -113,7 +113,7 @@ export class ActivityLogExpandedRowComponent
           },
           {
             label: $localize`FSP`,
-            value: item.attributes.fsp,
+            value: item.attributes.financialServiceProviderConfigurationLabel,
```

Comment:
is this fine? financialServiceProviderConfigurationLabel is a localizedString, whereas 'fsp' was not right?

Reply:
You are right :). 

The PR of dom is now merged in [feat.refactor-registration-data](https://github.com/global-121/121-platform/tree/feat.refactor-registration-data). So rebased this branch, so I could actually test the functionality, which now works